docs/products/SmartBear MCP Server/contract-testing-with-pactflow.md

```diff
@@ -2,24 +2,74 @@
 
 The PactFlow Contract testing client provides comprehensive tools which makes testing and deploying microservices at scale, simple and worry free for thousands of developers and testers around the world. Tools for PactFlow requires a `PACT_BROKER_BASE_URL` along with either a `PACT_BROKER_TOKEN` or (`PACT_BROKER_USERNAME` and `PACT_BROKER_PASSWORD`).
 
+Read more on PactFlow [Docs](https://docs.pactflow.io/).
 
 ## Available Tools
 
 ### Generate Pact Tests
 
-- Generate Pact tests using an OpenAPI spec in 6 different languages.
+- Purpose: Generate Pact tests from a number of inputs.
+- Returns: Generated Pact test
+- Parameters:
+  - **`language`** (optional)
+  The target language in which the Pact tests should be generated. If absent, the language will be inferred from other inputs, such as any provided code files. This field is optional if and only if it can be inferred from other inputs
```

Comment:
Last nit, but can you give this more breathing room? When rendered, it is very cramped. You can have paragraphs within dot points by having blank lines, and ensuring the next paragraph starts at the same indentation as the start of the text after the dot point marker:


```suggestion
  - **`language`** (optional)
    
    The target language in which the Pact tests should be generated. If absent, the language will be inferred from other inputs, such as any provided code files. This field is optional if and only if it can be inferred from other inputs
```